napari/_qt/qt_main_window.py

```diff
@@ -1031,13 +1075,23 @@ def activate(self):
         self._qt_window.raise_()  # for macOS
         self._qt_window.activateWindow()  # for Windows
 
+    def _update_window(self, event=None):
```

Comment:
@goanpeca do we need this function?  I commented it out and it doesn't seem necessary, but just want to make sure before I remove it, because I think you added it?

Reply:
Can't recall now, but we probably do not need it anymore